tests/Integration/TracerTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace DDTrace\Tests\Integration;
+
+use DDTrace\Tests\Unit\BaseTestCase;
+use DDTrace\Tracer;
+use DDTrace\Tests\Common\TracerTestTrait;
+use DDTrace\SpanData;
+
+final class TracerTest extends BaseTestCase
+{
+    use TracerTestTrait;
+
+    protected function setUp()
+    {
+        parent::setUp();
+        \putenv('DD_TRACE_GLOBAL_TAGS=global_tag:global');
+    }
+
+    protected function tearUp()
+    {
+        \putenv('DD_TRACE_GLOBAL_TAGS');
+        parent::tearDown();
+    }
+
+    public function testGlobalTagsArePresentOnLegacySpansByFlushTime()
+    {
+        $traces = $this->isolateTracer(function (Tracer $tracer) {
+            $scope = $tracer->startRootSpan('custom.name', [
+                'tags' => [
+                    'local_tag' => 'local',
+                ],
+            ]);
+            $scope->getSpan()->finish();
+        });
+
+        $this->assertSame('custom.name', $traces[0][0]['name']);
+        $this->assertSame('local', $traces[0][0]['meta']['local_tag']);
+        $this->assertSame('global', $traces[0][0]['meta']['global_tag']);
+    }
+
+    public function testGlobalTagsArePresentOnInternalSpansByFlushTime()
```

Comment:
I can see the test passed for this one on PHP 5.4. 😮But we should probably [skip this test on PHP 5.4](https://github.com/DataDog/dd-trace-php/blob/69b684121a4ddf67b79c5b24d452cfd7ba24e8e2/tests/Unit/TracerTest.php#L275-L278) since it does not support the sandbox API and might cause this test to become flaky.

Reply:
How is it possible that this test passes?

Reply:
Ah wait we [check for it PHP](https://github.com/DataDog/dd-trace-php/blob/116c107921376f6a3a1f2aea7fe4bbd00a9bd213/src/DDTrace/Configuration.php#L202), the way we do not use sandboxing in 5.4 is not loading sandboxed integrations in the loader. Now it makes sense.

Reply:
True, but since we're using `dd_trace_method()` directly, this test was still being run in PHP 5.4. Since the `dummyMethod()` doesn't have arguments or return a value, it will work as expected in PHP 5.4. But if there were arguments or a return value, things would not work as expected. In fact... I need to submit a PR to not expose `dd_trace_method()` and `dd_trace_function()` in PHP 5.4 until we can add support for that... 🤔